asteroid/scripts/asteroid_cli.py

```diff
@@ -90,25 +88,15 @@ def upload():
 
 
 def infer():
-    """CLI function to run pretrained model inference on wav files.
-
-    Args:
-        url_or_path(str): Path to the pretrained model.
-        files (List(str)): Path to the wav files to separate. Also support list
-            of filenames, directory names and globs.
-        force_overwrite (bool): Whether to overwrite output wav files.
-        resample (bool): Whether to resample wrong sample rate input files.
-        output_dir (str): Output directory to save files.
-    """
-    import argparse
-
+    """CLI function to run pretrained model inference on wav files."""
     parser = argparse.ArgumentParser()
     parser.add_argument("url_or_path", type=str, help="Path to the pretrained model.")
```

Comment:
Unrelated to this PR, but I think this is not very good design. You will always pass exactly 1 model, but N files. Usually you'd use a `--named-flag` for the model here, and unnamed/positional arguments for the files: `asteroid-infer -m mymodel file1.wav file2.wav ...`.

Reply:
Ok, thanks. I haven't been using UNIX for so long so I don't know much.
Note that we support globs in --files, but I understand the point.